assets/js/builder/Schemas/Quiz.js

```diff
@@ -67,8 +69,20 @@ define( [], function() {
 						label: LLMS.l10n.translate( 'Randomize Question Order' ),
 						tip: LLMS.l10n.translate( 'Display questions in a random order for each attempt. Content questions are locked into their defined positions.' ),
 						type: 'switch',
+						condition: function() {
+							return 'yes' === this.get( 'question_bank' ) ? false : true;
+						}
 			},
-				],
+				], [
+					{
+						id: 'question-bank',
+						label: LLMS.l10n.translate( 'Question Bank' ),
+						tip: LLMS.l10n.translate( 'A question bank helps prevent cheating by allowing instructors to create assessments with randomized questions pulled from a bank of questions. (Available in Advanced Quizzes addon)' ),
+						type: 'disabled',
+						text: 'Get LifterLMS Advanced Quizzes',
+						visibility: window.llms.hooks.applyFilters( 'llms_qb_upsell_visibility', 'visible' ),
```

Comment:
Instead of playing with this filter (hence on the field visibility), can't we just leverage the filter `llms_define_quiz_schema`, and replace this field with the proper one from the add-on?
This will also avoid the addition of the `visibility` at all. Right?

Reply:
Nice suggestion 💯 , thanks. Fixed it.